src/EPR.RegulatorService.Frontend.Web/Program.cs

```diff
@@ -46,6 +45,10 @@
     .AddApplicationInsightsTelemetry()
     .AddHealthChecks();
 
+builder.Logging.ClearProviders();
+builder.Logging.AddConsole();
```

Comment:
B2C is _incredibly_ chatty on this service; we may need to explicitly specify a log level filter.